nchelpers/__init__.py

```diff
@@ -33,6 +72,26 @@ def standard_climo_periods(calendar='standard'):
             for k, (start_year, end_year) in standard_climo_years.items()}
 
 
+def _join_comma_separated_list(s, sep='+'):
```

Comment:
Maybe name this ``_replace_commas``? You don't use ``join()`` and there are no lists anywhere in the function, so it's a little bit of a misnomer.